tables/basic-table/step3.md

```diff
@@ -0,0 +1,64 @@
+Now that you have the basic table structure and imports in place, let's express column and row data as objects. This will allow us to attach and adjust data about individual rows/cells/columns later on.
+
+## Task
+
+1) Locate the following code `const columns = [...]`
+
+2) For each of the column headers in the array, replace the string with an object and set the object’s title property to the string value you had previously.
+
+For example: `"First column"` becomes `{ title: "First column" }`
+
+2) Now apply this same type of transformation to each of your row definitions.
+
+You're ultimately replacing the string array with an array of objects, where the cell property of the object is the row array you had previously.
+
+For example: `["Row 1 column 1", "Row 1 column 2", "Row 1 column 3"]` becomes
+
+<pre class="file">
+{
+  cells: ["Row 1 column 1", "Row 1 column 2", "Row 1 column 3"]
+}
+</pre>
```

Comment:
Yes definitely can, when I was putting some of these steps together I intentionally omitted the copy for a few of the bits to encourage people to try typing it out (you can still click+drag to highlight a section and copy that way also). I can see how it will help move things along and make the experience consistent across many tutorials though, so I'll add them to all the snippets except maybe the few places where I show the final implementation at the end?